tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -898,7 +940,7 @@ async function ensureInClusterLegacyEventReceivedWithRetry(mockHost, eventId, ev
 
     if (eventType) {
       debug(`checking if received event type is: ${eventType}`);
-      expect(response.data).to.have.nested.property('event.ce-type').that.contains(eventType);
+      expect(response.data).to.have.nested.property('event.type').that.contains(eventType);
```

Comment:
Why it is not `ce-type`? We receive a cloud event, there should be `ce-type` header??

Reply:
the is done [here](https://github.com/kyma-project/kyma/blob/d984ba1a9a1c1b735d8c0f7d6d5fbccb0b64f4a0/tests/fast-integration/test/fixtures/commerce-mock/index.js#L906) too. It really didn't work out with `ce-type`. I could spend more time and fix it. Let's discuss if you think it is not right check.